tests/Transport/NodePool/TraceableSimpleNodePool.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Elastica\Test\Transport\NodePool;
+
+use Elastic\Transport\NodePool\Node;
+use Elastic\Transport\NodePool\SimpleNodePool;
+
+final class TraceableSimpleNodePool extends SimpleNodePool
```

Comment:
We need this for testing purposes as original node pool doesn't have `getNodes` method so we couldn't check different cases in our tests